backend/backend/settings.py

```diff
@@ -1,3 +1,4 @@
+import django_heroku
```

Comment:
This import actually needs to be at the bottom of the file, for unclear reasons. Please move it back there.

Reply:
vs code automatically moves all imports to the top when I save.

Reply:
Hmmm...you may need to turn off autoformatting, then, as it shouldn't be doing that.